include/aktualizr-lite/api.h

```diff
@@ -157,13 +182,21 @@ class AkliteClient {
    */
   bool IsRollback(const TufTarget &t) const;
 
+  /**
+   * Set the secondary ECUs managed by this device. Will update the status of
+   * the ECUs on the device-gateway and instruct the CheckIn method to also
+   * look for targets with the given hardware ids.
+   */
+  InstallResult SetSecondaries(const std::map<std::string, SecondaryEcu> &ecus);
```

Comment:
What does the map key mean/denotes, I mean the first element of the map?

Reply:
After looking into it for the second time, I think it would have made the code slightly more readable if `SecondaryEcu` had included a `serial` field and `std::vector<SecondaryEcu>` had been passed as a param.

Reply:
I've force pushed a new revision that does this.

I *think* this is what we've accepted for now that can be merged